ci/integration_tests/src/helpers/helper.rs

```diff
@@ -1,14 +1,14 @@
 use super::chain::Chain;
 use anyhow::Result;
-use cosmwasm_std::{to_binary, Decimal, Uint128};
+use cosmwasm_std::{to_binary, Decimal, Empty, Uint128};
 use cw20::Cw20Coin;
-use cw_core::{
-    msg::{Admin, ModuleInstantiateInfo},
-    query::DumpStateResponse,
-};
+use cw_core::query::DumpStateResponse;
```

Comment:
should the pre-propose module be included in cw-core's DumpState?

Reply:
decided it would be better to keep as is and allow the frontend to query the proposal module's proposal_creation_policy instead.. less complicated and handles case where creation policy is Anyone